File: OLDNAPAKALAKI/NapakalakiRuby/lib/prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
#encoding: utf-8

class Prize
  def initialize(treasure, level)
    @treasures = treasure
    @level = level
  end
  
  attr_accessor :trasures, :level
  
def to_s
  "\n    Tesoros ganados: #{@treasures}\n    Niveles ganados: #{@level}\n"
end
  
end

File: teoria/examen1/Napakalaki/src/napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author Redondo
 */
public class Prize {
    private int treasure;
    private int level; 

    public Prize(int treasure, int level) {
        this.treasure = treasure;
        this.level = level;
    }

    public int getTreasure() {
        return treasure;
    }

    public int getLevel() {
        return level;
    }

    
    @Override
    public String toString() {
        return "Prize{" + "treasure=" + treasure + ", level=" + level + '}';
    }
    
       
    
}
